README.org

```diff
@@ -1,5 +1,13 @@
+
 [[https://melpa.org/#/citar][file:https://melpa.org/packages/citar-badge.svg]]
 
+* *!!! BREAKING CHANGE NOTICE !!!*
```

Comment:
I don't feel strongly about it, but you could just have it "Breaking Changes", and then have an itemized list (of one currently).